src/util/loadLocalState.js

```diff
@@ -62,7 +55,6 @@ export const loadLocalState = async () => {
     []
   )
 
-  // if localForage has data but schemaVersion is not defined, it means we are at the SCHEMA_HASHKEYS version
```

Comment:
Rather than delete the whole comment, just replace `localForage` with `local database`. The logic is still true here.